common/djangoapps/enrollment/tests/test_views.py

```diff
@@ -32,6 +32,15 @@
 from embargo.test_utils import restrict_course
 
 
+class UrlMixin(object):
+    """
+    Mixins for URL processing
+    """
+    def get_absolute_url(self, path):
+        """ Generate an absolute URL for a resource on the test server. """
+        return u'http://testserver/{}'.format(path.lstrip('/'))
+
+
```

Comment:
I think I'm just disagreeing that a mixin is the right way to share this functionality.  If there's no builtin django function that already does what this function is doing, then a static utility function would be sufficient, since `self` is never referenced.  The advantage of not using a mixin is simply less code and more straightforward readability.   Also, as it stands, users of this mixin need to be aware that the prefix "http://testserver/" is assumed, which seems like a gotcha to me.